nc-news_/src/Components/SingleArticle/SingleArticle.jsx

```diff
@@ -0,0 +1,10 @@
+import { useParams } from "react-router";
+import ArticleBody from "./ArticleBody";
+
+const SingleArticle = () => {
+  const { id } = useParams();
+  
+  return <ArticleBody id={id} />;
+};
```

Comment:
There's no need for the `SingleArticle` component if its only job is to grab the id and then send to another component. My suggestion is to move everything from the `ArticleBody` component to this one and then render this SingleArticle component in your route instead

Reply:
I did it that way because I plan to have the comments be their own component as a child of single article as the only thing they share with the article body is the id